### PR TITLE
add circular button as variant to button component

### PR DESCRIPTION
**Pull request by wytseng**

### CODE REVIEW

File: app/src/components/ui/Button.tsx [hunk 1]
```diff
@@ -24,6 +24,10 @@ const buttonStyles = cva(
       fullWidth: {
         true: "w-full",
       },
+      design: {
+        // for icon only buttons (icon has to be square shaped)
+        circular: "p-2.5 rounded-full",
```
Comment (lsuan) [inline app/src/components/ui/Button.tsx:29]: oh `p-2.5` works?

Comment (wytseng) [reply app/src/components/ui/Button.tsx:29]: Ye it's should be 10px according to tailwind and works for me. I referenced Lindsey's figma design cuz the padding was 10px

Comment (wytseng) [reply app/src/components/ui/Button.tsx:29]: image
lmk if you want me to change it, but p-2.5 does give 10px padding

Comment (lsuan) [reply app/src/components/ui/Button.tsx:29]: ooo okay, this is fine !

**Review by lsuan — APPROVED** [review]

**Pull request merged**